src/Plugin_Language_Command.php

```diff
@@ -318,10 +324,10 @@ private function install_many( $args, $assoc_args ) {
 			$available = $this->get_installed_languages( $plugin_name );
 
 			foreach ( $language_codes as $language_code ) {
-				$result = [
+				$result = array(
 					'name'   => $plugin_name,
 					'locale' => $language_code,
-				];
+				);
```

Comment:
There is an unwanted change here (probably automated tooling change). WP-CLI uses short array syntax.

Reply:
Fixed in d54ebb2f